comps/llms/faq-generation/vllm/langchain/README.md

```diff
@@ -0,0 +1,75 @@
+# vLLM FAQGen LLM Microservice
+
+This microservice interacts with the vLLM server to generate FAQs from Input Text.[Text Generation Inference](https://github.com/huggingface/text-generation-inference) (vLLM) is a toolkit for deploying and serving Large Language Models (LLMs). VLLM enables high-performance text generation for the most popular open-source LLMs, including Llama, Falcon, StarCoder, BLOOM, GPT-NeoX, and more.
```

Comment:
This needs to be changed to point to VLLM?

Reply:
Updated it